src/routes/email-grid.ts

```diff
@@ -0,0 +1,26 @@
+import { Request, Response, Router } from 'express';
+import { Mail } from '../types/mail';
+import { sendEmail } from '../utils/mail/sendgrid';
+
+export const emailGridRouter = Router();
+
+/**
+ * @description POST request to send an email from an authorized Gmail account
+ */
+emailGridRouter.post('/send-grid', async (req: Request, res: Response) => {
```

Comment:
Lets set this to just `emailGridRouter.post('/',...` generally its better practice to keep the RESTful API structure as more of a black box, the client isn't concerned with if the email is sent using SendGrid or Gmail so abstraction here is good.